examples/metaobjects/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "metaobjects",
+  "private": true,
+  "sideEffects": false,
+  "version": "1.0.0",
+  "scripts": {
+    "build": "shopify hydrogen build",
+    "dev": "shopify hydrogen dev --codegen",
+    "preview": "npm run build && shopify hydrogen preview",
+    "lint": "eslint --no-error-on-unmatched-pattern --ext .js,.ts,.jsx,.tsx .",
+    "typecheck": "tsc --noEmit",
+    "codegen": "shopify hydrogen codegen"
+  },
+  "prettier": "@shopify/prettier-config",
+  "dependencies": {
+    "@remix-run/react": "^2.5.1",
+    "@shopify/cli": "3.51.0",
+    "@shopify/cli-hydrogen": "^7.0.0",
+    "@shopify/hydrogen": "^2024.1.0",
+    "@shopify/remix-oxygen": "^2.0.2",
```

Comment:
It looks like [each example is introducing different dependency versions](https://github.com/Shopify/hydrogen/pull/1715/commits/f9c0d603322ba50e65a4194531ee1ef904090717) from what we use in the rest of the workspace. I'm fixing that separately in another branch so let's avoid the problem here as well.

Alternatively, use `--diff` to avoid listing repeated dependencies in examples.

```suggestion
    "@shopify/cli": "3.52.0",
    "@shopify/cli-hydrogen": "^7.0.0",
    "@shopify/hydrogen": "~2024.1.0",
    "@shopify/remix-oxygen": "^2.0.3",
```